composer/callbacks/__init__.py

```diff
@@ -1,16 +1,12 @@
 # Copyright 2021 MosaicML. All Rights Reserved.
 
-"""Callbacks provide hooks that can run at each training loop :class:`~composer.core.event.Event`
+"""Callbacks that run at each training loop :class:`~composer.core.event.Event`.
 
 Each callback inherits from the :class:`~composer.core.callback.Callback` base class. See detailed description and
 examples for writing your own callbacks at the :class:`~composer.core.callback.Callback` base class.
 """
-from composer.callbacks.callback_hparams import CallbackHparams as CallbackHparams
-from composer.callbacks.callback_hparams import GradMonitorHparams as GradMonitorHparams
-from composer.callbacks.callback_hparams import LRMonitorHparams as LRMonitorHparams
-from composer.callbacks.callback_hparams import MemoryMonitorHparams as MemoryMonitorHparams
-from composer.callbacks.callback_hparams import RunDirectoryUploaderHparams as RunDirectoryUploaderHparams
-from composer.callbacks.callback_hparams import SpeedMonitorHparams as SpeedMonitorHparams
+from composer.callbacks.callback_hparams import (CallbackHparams, GradMonitorHparams, LRMonitorHparams,
+                                                 MemoryMonitorHparams, RunDirectoryUploaderHparams, SpeedMonitorHparams)
```

Comment:
Should we use the same pattern at other places ? For example, I was working on composer.datasets documentation and was wondering If I should use this same pattern there.

Reply:
Yes, @ravi-mosaicml said how the `loggers` init is setup is the correct approach.